src/core/configuration/configurationService.ts

```diff
@@ -38,6 +39,20 @@ class ConfigurationService {
     return this.configurationEnv;
   }
 
+  private setKeriaIp() {
+    const keriaUrl = ConfigurationService.configurationEnv.keri.keria.url;
+    const keriaBootUrl =
+      ConfigurationService.configurationEnv.keri.keria.bootUrl;
+    if (keriaIP) {
+      ConfigurationService.configurationEnv.keri.keria.url = keriaUrl.replace(
+        /\/\/[^:]+/,
```

Comment:
Does this work with just IPs or also hostnames? In case someone has localhost. (though localhost AFAIK doesn't work with KERIA right now, but in the future it might)

Reply:
Yes, It work with localhost.